Infrastructure/PluginExecutor.cs

```diff
@@ -42,16 +49,24 @@ public void Run(string message, string user)
                     || CheckIfIlegalPluginPressed(message, out int pluginNumber, out msgForUser))
                 {
                     _messageSender.SendMessage(user, msgForUser);
-                    return;
+                    return ;
```

Comment:
Here you added an extra space, get use to format the code...

Reply:
OK